src/Instrumentation/HttpAsyncClient/_register.php

```diff
@@ -2,4 +2,11 @@
 
 declare(strict_types=1);
 
-\OpenTelemetry\Contrib\Instrumentation\HttpAsyncClient\HttpAsyncClientInstrumentation::register();
+use OpenTelemetry\Contrib\Instrumentation\HttpAsyncClient\HttpAsyncClientInstrumentation;
+use OpenTelemetry\SDK\Sdk;
+
+if (class_exists(Sdk::class) && Sdk::isInstrumentationDisabled(HttpAsyncClientInstrumentation::NAME) === true) {
```

Comment:
what happens here if no SDK and no extension? we likely want another condition for "not extension installed"?

Reply:
Good spot, the warning was not added here yet. I have added it here too